Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: clean deps fmt vet test docker
+
+EXECUTABLE ?= drone-pypi
+IMAGE ?= plugins/$(EXECUTABLE)
+CI_BUILD_NUMBER ?= 0
+
+LDFLAGS = -X "main.buildDate=$(shell date -u '+%Y-%m-%d %H:%M:%S %Z')"
+PACKAGES = $(shell go list ./... | grep -v /vendor/)
+
+clean:
```

Comment:
Maybe add something like this as the first rule:

```
all: build
```

Having `clean` as the default target seems unconventional and a bit confusing.